src/components/form/Form.js

```diff
@@ -1,13 +1,25 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { reduxForm } from 'redux-form';
+import { compose } from 'redux';
+import { connect } from 'react-redux';
+
+import * as actions from '@/actions/creators';
 
 import FormFieldBuilder from './FormFieldBuilder';
 
+const mapDispatchToProps = (dispatch, ownProps) => ({
+  onSave: (e) => {
```

Comment:
I'd use the other way of writing, like so:
```
onSave(e) {
},
```

Reply:
in mapDispatchToProps I wouldn't so we keep the same form as one-line functions